R/pkg/inst/tests/testthat/test_Windows.R

```diff
@@ -20,7 +20,7 @@ test_that("sparkJars tag in SparkContext", {
   if (.Platform$OS.type != "windows") {
     skip("This test is only for Windows, skipped")
   }
-  testOutput <- launchScript("ECHO", "a/b/c", capture = TRUE)
+  testOutput <- launchScript("ECHO", "a/b/c", wait = TRUE)
```

Comment:
I see - I expected the stdout = "" to be piping it to the R process stdout. We could also explicitly pass a `fd` to do this pipe ?